src/betterTableLayout.js

```diff
@@ -21,60 +103,53 @@ const betterTableLayout = () => {
         const date = row.find('td:nth-child(3)').clone(true, true);
         const weight = row.find('td:nth-child(4)').clone(true, true);
         const denominator = $(this).find('td:nth-child(5)').clone(true, true);
+
+        name.css('width', '');
```

Comment:
```suggestion
```
This line is useless

Reply:
I'm pretty sure it does something. Since Maplewood sets every name column cell to have a fixed width of 300px, in order for the percentage styling to take effect, the width has to be cleared.